test/support.js

```diff
@@ -247,15 +248,22 @@ const Support = {
       }
     }
 
+    if (query instanceof Composition) {
+      query = Support.sequelize.dialect.QueryGenerator.composeQuery(query);
+    }
+
     if (query instanceof Error) {
       expect(query.message).to.equal(expectation.message);
     } else {
       expect(query.query || query).to.equal(expectation);
-    }
 
-    if (assertions.bind) {
-      const bind = assertions.bind[Support.sequelize.dialect.name] || assertions.bind['default'] || assertions.bind;
-      expect(query.bind).to.deep.equal(bind);
+      if (assertions.bind) {
+        let bind = assertions.bind[Support.sequelize.dialect.name] || assertions.bind['default'] || assertions.bind;
+        if (Support.sequelize.dialect.name === 'mssql' && Array.isArray(bind)) {
```

Comment:
Note: I know this is a common (anti)pattern, but shouldn't we be checking `dialect.supports.*` instead?

Reply:
I changed my mind, for simplicity I will call `outputBind` for every dialect, which should translate regular bind arrays to specific dialect formats(if any).